UI/scripts/main.js

```diff
@@ -4,17 +4,17 @@
 /* eslint-disable func-names */
```

Comment:
Remove the eslint disable lines

Reply:
I have done that in a separate branch by addding the .eslintrc file you recommended. But do I add the file to .gitignore?